aut_docs/Documented_Failures.md

```diff
@@ -0,0 +1,12 @@
+There are a few tests which have been commented out currently to avoid travis build failure:
```

Comment:
Capitalize Travis. Replace with Travis CI, maybe. This is an official doc. You need it to be as perfect as possible.